WORKSPACE

```diff
@@ -18,14 +18,14 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 daml_deps()
 
-load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
-
-protobuf_deps()
-
 load("@rules_haskell//haskell:repositories.bzl", "rules_haskell_dependencies")
 
 rules_haskell_dependencies()
 
+load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
+
+protobuf_deps()
+
```

Comment:
Did you move it for a specific reason?

Reply:
yeah it is a bazel-fu workaround as advised by @aherrmann https://github.com/digital-asset/daml/pull/6063#issuecomment-633878929